docker/changeDatabase.xsl

```diff
@@ -1,48 +1,54 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <xsl:stylesheet version="2.0"
-                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
-                xmlns:ds="urn:jboss:domain:datasources:4.0">
```

Comment:
That's my IDEA formatter messing around - shall I revert back to the original one?